src/test/resources/features/cards/createCard.feature

```diff
@@ -14,7 +14,9 @@ Feature: Card
     Given I log in with my Trello account as "user1"
     And I navigate to boards menu from header
     And I open the "New board" board
+#    When I create a Card on "Jacoco" list
```

Comment:
Lets errase this comentaries.

Reply:
I think you need to apply this changes, in order to solve our trainer observations.